File: ws/aymacana/u1/ws04Dependencies/ChickenFarmSimulator/src/ec/edu/espe/chickenfarmsimulator/model/Egg.java
package ec.edu.espe.chickenfarmsimulator.model;

/**
 *
 * @author LABS-ESPE
 */
public class Egg {

    private char size;

    @Override
    public String toString() {
        return "Egg{" + "size=" + size + '}';
    }
    
    public Egg(char size) {
        this.size = size;
    }

    /**
     * @return the size
     */
    public char getSize() {
        return size;
    }

    /**
     * @param size the size to set
     */
    public void setSize(char size) {
        this.size = size;
    }

}

File: hw/calle/u1/hw11ChikenCoops/ChickenCoopsJavaScript/Egg.js
class Egg {
  /**
   * @param {string} size
   */
  constructor(size) {
    this.size = size;
  }

  toString() {
    return `Egg{ size = ${this.size} }`;
  }

  getSize() {
    return this.size;
  }
}

module.exports = { Egg };